android/gradle.properties

```diff
@@ -1 +1,2 @@
 org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
+AsyncStorage_db_size_in_MB=60
```

Comment:
are you sure this is enough?

Reply:
I would say 6 MB is enough, but 10 x 'enough' = more than enough ;D